lib/debuglet.js

```diff
@@ -333,27 +333,23 @@ Debuglet.prototype.addBreakpoint_ = function(breakpoint, cb) {
     that.logger_.info('\tsuccessfully added breakpoint  ' + breakpoint.id);
     that.activeBreakpointMap_[breakpoint.id] = breakpoint;
 
-    that.v8debug_.wait(breakpoint, function waitHandler(err) {
-      if (err) {
-        that.logger_.error(err);
-        cb(err);
-        return;
-      }
-
-      that.logger_.info('Breakpoint hit!: ' + breakpoint.id);
-      if (breakpoint.action === 'LOG') {
-        var message = Debuglet.format(breakpoint.logMessageFormat,
-          breakpoint.evaluatedExpressions.map(JSON.stringify));
-        console.log(message);
-        if (!that.completedBreakpointMap_[breakpoint.id]) {
-          setTimeout(function() {
-            that.v8debug_.wait(breakpoint, waitHandler);
-          }, 500);
+    if (breakpoint.action === 'LOG') {
+      that.v8debug_.log(breakpoint, Debuglet.format,
+        function() {
+          return that.completedBreakpointMap_[breakpoint.id];
+        });
+    } else {
+      that.v8debug_.wait(breakpoint, function(err) {
```

Comment:
It was previously referenced and this change removes the need to reference it. The name isn't hurting but we generally don't name unreferenced callbacks so I removed it.